modules/git/repo_branch.go

```diff
@@ -13,6 +13,14 @@ import (
 // BranchPrefix base dir of the branch information file store on git
 const BranchPrefix = "refs/heads/"
 
+// AGit Flow
+
+// PullRequestPrefix sepcial ref to create a pull request: refs/for/<targe-branch>/<topic-branch>
+// or refs/for/<targe-branch> -o topic='<topic-branch>'
+const PullRequestPrefix = "refs/for/"
```

Comment:
can you move this const into **services/agit/agit.go** ?

Reply:
never mind - just checked how other refs prefix are handled and it need a refactor ...
-> I propose to move things into modules/git/ref.go and replace strings in code with that const ... witch is worth it's own pull